### DISCUSSION TITLE
Kernel, Stride combos that work

### DISCUSSION THREAD

**Discussion by ebgoldstein** [Doodleverse/segmentation_gym#47] [Ideas]
does it matter with target size?

- [ ] add to readme
- [ ] come up with solution (script?).. this is a fn of upsampling2d

**Comment by ddbuscombe** [comment]
what i know....

kernel > 1
target size must be even, therefore upsampling has to be 2x2 (see original paper), but this makes some kernel/strides combos impossible

**Comment by ddbuscombe** [comment]
I also think there may be fewer options when using deconvolution for upsampling

**Comment by ddbuscombe** [comment]
Perhaps this will help https://explained.ai/tensor-sensor/index.html

**Comment by ddbuscombe** [comment]
Stride must be 2. Next!
